File: BasicSampleJava/app/src/main/java/com/regula/facesample/items/customization/fragment/NotificationViewFragment.java
package com.regula.facesample.items.customization.fragment;

import android.content.Context;
import android.graphics.drawable.Drawable;

import androidx.core.content.ContextCompat;

import com.regula.facesample.R;
import com.regula.facesdk.fragment.FaceDefaultUiFragment;

/**
 * Created by Sergey Yakimchik on 6.09.21.
 * Copyright (c) 2021 Regula. All rights reserved.
 */

public class NotificationViewFragment extends FaceDefaultUiFragment {

    @Override
    public Drawable getNotificationBackgroundDrawable(Context context, boolean isLightOn) {
        return ContextCompat.getDrawable(context, isLightOn ? R.drawable.notification_view_background_dark : R.drawable.notification_view_background_white);
    }

    @Override
    public int getNotificationTextColor(boolean isLightOn) {
        return getResources().getColor(isLightOn ? R.color.notification_view_text_color_light_on : R.color.notification_view_text_color_light_off);
    }
}

File: FaceSampleKotlin/app/src/main/java/com/regula/facesamplekotlin/fragments/NotificationViewFragment.kt
package com.regula.facesamplekotlin.fragments

import android.content.Context
import android.graphics.drawable.Drawable
import androidx.core.content.ContextCompat
import androidx.core.content.res.ResourcesCompat
import com.regula.facesamplekotlin.R
import com.regula.facesdk.fragment.FaceDefaultUiFragment

class NotificationViewFragment : FaceDefaultUiFragment() {

    override fun getNotificationBackgroundDrawable(
        context: Context?,
        isLightOn: Boolean
    ): Drawable? {
        return context?.let {
            ContextCompat.getDrawable(
                it, if (isLightOn) R.drawable.notification_view_background_dark else R.drawable.notification_view_background_white
            )
        }
    }

    override fun getNotificationTextColor(isLightOn: Boolean): Int {
        return ResourcesCompat.getColor(resources, if (isLightOn) R.color.notification_view_text_color_light_on else R.color.notification_view_text_color_light_off, null)
    }
}